### ISSUE TITLE
Nothing Showing on Overviewer load

### ISSUE THREAD

**Issue by Electromaster232** [overviewer/Minecraft-Overviewer#1421]
Hi all!
I've been having an issue making this work. After editing index.html to add my API key, it seems my world does not load.
https://theendlessweb.com/mcmap
Using Ubuntu 16.04.03 LTS
Nginx as my server

Any thoughts?

(P.S. I know the world is not corrupted, I built a PNG version which you can see: https://theendlessweb.com/map3)

**Comment by Gerolfing** [comment]
Looks like I am having same troubles. Everything was working fine and today I got same issue. (Also Ubuntu 16.04)

**Comment by Gerolfing** [comment]
Looks like map problem, but still I don't get it where that last region file came from
![wat](https://user-images.githubusercontent.com/23654416/34034590-d5083b3a-e187-11e7-9ddd-20367448460f.png)
...

**Comment by Electromaster232** [comment]
not sure. It could just be a ubuntu thing in general, I tried some different world renderers (including DynMap) none of which worked at all.

**Comment by Gerolfing** [comment]
I fixed it by running /wb trim command, which deleted everything beyond worldborder. I suggest you to check if you have distant chunks on map that you try to render.

**Comment by Electromaster232** [comment]
So to bring this out of the issue graveyard, i'm going to try something. I took a very small spigot test server, and ran DynMap (with the same broken map I used above) and guess what! It worked perfectly. Note that I did run this on my very powerful gaming PC, so that might be the reason it worked, but considering Overviewer builds static files, isn't to much of a pain.

tl;dr im going to try running Overviewer on here now and see if I get any results.

**Comment by Gerolfing** [comment]
Do you have far away map chunks? Because they were the reason of black render pictures in my case. After first time I had 2 more such issues and every time I deleted chunks beyond border with worldborder plugin (/wb trim). Then I deleted all corrupted files from overviewer render folder and after full render everything worked as usual.

**Comment by Electromaster232** [comment]
I might try that before anything else 👌
